mock/py/mockbuild/config.py

```diff
@@ -50,7 +50,8 @@ def nspawn_supported():
 @traceLog()
 def setup_default_config_opts(unprivUid, version, pkgpythondir):
     "sets up default configuration."
-    config_opts = TemplatedDictionary(alias_spec={'dnf.conf': ['yum.conf']})
+    config_opts = TemplatedDictionary(alias_spec={'dnf.conf': ['yum.conf'],
+                                                  'dnf_vars': ['yum_vars']})
```

Comment:
The movement being done now is that mock v3 doesn't have to be compatible with yum at all.
We should start removing yum-compat hacks, rather than add new ones.

Reply:
We still need to be able to build for targets that use yum, even if the host does not.

Reply:
I personally don't think we need this rather risky change which fixes a cosmetic issue.

Reply:
This is not a cosmetic issue. Please read again:
https://github.com/rpm-software-management/mock/pull/809#issuecomment-979231466

Because the same thing applies to yum.

Reply:
The fact we have it expanded is a cosmetic issue.

Reply:
You're missing this part:
> If users craft a custom config they may use it.

Reply:
I got your point, and I agree.  Some users might want to use this in the custom configs.